### PR TITLE
intro&deploy-nebula-operator-en

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# What is Nebula Operator
+
+## Concept of Nebula Operator
+
+Nebula Operator is a tool to automate the deployment, and operation and maintenance of [Nebula Graph](https://github.com/vesoft-inc/nebula) clusters on [Kubernetes](https://kubernetes.io). Building upon the excellent scalability mechanism of Kubernetes, Nebula Graph introduced its operation and maintenance knowledge into the Kubernetes system, which makes Nebula Graph a real [cloud-native graph database](https://www.nebula-cloud.io/).
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:5]:
```suggestion
Nebula Operator is a tool to automate the deployment, operation, and maintenance of [Nebula Graph](https://github.com/vesoft-inc/nebula) clusters on [Kubernetes](https://kubernetes.io). Building upon the excellent scalability mechanism of Kubernetes, Nebula Graph introduced its operation and maintenance knowledge into the Kubernetes system, which makes Nebula Graph a real [cloud-native graph database](https://www.nebula-cloud.io/).
```

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## How it works
+
+For resource types that do not exist within Kubernetes，you can register them by adding custom API objects. The common way is to use the [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions).
+
+Nebula Operator abstracts the deployment management of Nebula Graph clusters as a CRD. By combining multiple built-in API objects including StatefulSet, Service, and ConfigMap, the routine management and maintenance of a Nebula Graph cluster are coded as a control loop in the Kubernetes system. When a CR instance is submitted, Nebula Operator drives database clusters to the final state according to the control process.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:11]:
```suggestion
Nebula Operator abstracts the deployment management of Nebula Graph clusters as a CRD. By combining multiple built-in API objects including StatefulSet, Service, and ConfigMap, the routine management, and maintenance of a Nebula Graph cluster are coded as a control loop in the Kubernetes system. When a CR instance is submitted, Nebula Operator drives database clusters to the final state according to the control process.
```

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Functionalities of Nebula Operator
+
+The following features are already available in Nebula Operator:
+
+- **Deploy and uninstall clusters**：Nebula Operator simplifies the process of deploying and uninstalling clusters for users. Nebula Operator allows you to quickly create, update, or delete a Nebula Graph cluster by simply providing the corresponding CR file. For more information, see [Deploy Nebula Graph Clusters with Kubectl](3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md) or [Deploy Nebula Graph Clusters with Helm](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md).
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:17]:
```suggestion
- **Deploy and uninstall clusters**: Nebula Operator simplifies the process of deploying and uninstalling clusters for users. Nebula Operator allows you to quickly create, update, or delete a Nebula Graph cluster by simply providing the corresponding CR file. For more information, see [Deploy Nebula Graph Clusters with Kubectl](3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md) or [Deploy Nebula Graph Clusters with Helm](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md).
```

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  
+- **Scale clusters**：Nebula Operator calls Nebula Graph's native scaling interfaces in a control loop to implement the scaling logic. You can simply perform scaling operations with YAML configurations and ensure the stability of data. For more information, see [Scale clusters with Kubectl](3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md#_3) or [Scale clusters with Helm](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md#_2).
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:19]:
```suggestion
- **Scale clusters**: Nebula Operator calls Nebula Graph's native scaling interfaces in a control loop to implement the scaling logic. You can simply perform scaling operations with YAML configurations and ensure the stability of data. For more information, see [Scale clusters with Kubectl](3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md#_3) or [Scale clusters with Helm](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md#_2).
```

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  
+- **Self-Healing**：Nebula Operator calls the interface provided by Nebula Graph clusters to dynamically sense cluster service status. Once an exception is detected, Nebula Operator performs fault tolerance. For more information, see [Self-Healing](5.operator-failover.md).
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:21]:
```suggestion
- **Self-Healing**: Nebula Operator calls the interface provided by Nebula Graph clusters to dynamically sense cluster service status. Once an exception is detected, Nebula Operator performs fault tolerance. For more information, see [Self-Healing](5.operator-failover.md).
```

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  
+- **Balance Scheduling**: Based on the scheduler extension interface, the scheduler provided by Nebula Operator evenly distributes Pods in a Nebula Graph cluster across all nodes.
+
+## Limitations
+
+### Version limitations
+
+Nebula Operator does not support v1.x version of Nebula Graph. Nebula Operator version and the corresponding Nebula Graph version are as follows:
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:29]:
```suggestion
Nebula Operator does not support the 1.x version of Nebula Graph. Nebula Operator version and the corresponding Nebula Graph version are as follows:
```

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+| Nebula Operator version | Nebula Graph version |
+| ------------------- | ---------------- |
+| {{operator.release}}| {{nebula.release}} |
+
+### Functionality limitations
+
+Nebula Operator currently only support manual scaling of Nebula Graph clusters, and does not support automatic scaling of Nebula Graph clusters.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:37]:
```suggestion
Nebula Operator currently only supports manual scaling of Nebula Graph clusters, and does not support automatic scaling of Nebula Graph clusters.
```

Comment (abby-cyber) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:37]: Nebula Operator currently only supports manual

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md (hunk 5 above)
Comment (randomJoe211) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:21]: The paragraph above said that "Nebula Graph's native scaling interfaces", but here you used "the interface provided by Nebula Graph clusters". I guess the latter sentence should use the plural form as well?

File: docs-2.0/nebula-operator/2.deploy-nebula-operator.md [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+| `imagePullSecrets`                     | -                               | The image pull secret in Kubernetes.                            |
+| `kubernetesClusterDomain`              | `cluster.local`。           | The cluster domain.                                |
+| `controllerManager.create`             | `true`                          | Whether to enable the controller-manager component.              |
+| `controllerManager.replicas`           | `2`                             | The numeric value of controller-manager replicas.                |
+| `admissionWebhook.create`              | `true`                          | Whether to enable Admission Webhook.               |
+| `shceduler.create`                     | `true`                          | Whether to enable Scheduler.                       |
+| `shceduler.schedulerName`              | `nebula-scheduler`              | The Scheduler name.                              |
+| `shceduler.replicas`                   | `2`                             | the numeric value of nebula-scheduler replicas.                  |
```
Comment (izhuxiaoqing) [inline docs-2.0/nebula-operator/2.deploy-nebula-operator.md:145]:
```suggestion
| `shceduler.replicas`                   | `2`                             | The numeric value of nebula-scheduler replicas.                  |
```

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md (hunk 2 above)
Comment (abby-cyber) [reply docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:11]: the subject is maintenance and management, will keep the original content.

**Review by cooper-lzy — APPROVED** [review]

**Review by foesa-yang — APPROVED** [review]

**Pull request merged**